src/components/alerts/alert.css

```diff
@@ -0,0 +1,29 @@
+@import "../../css/units.css";
+@import "../../css/colors.css";
+@import "../../css/z-index.css";
+@import "../gui/gui.css";
```

Comment:
This felt a little weird, but seems to be the only way to get the alertsContainer style in gui.css to be accessible in the alert component?

Reply:
To share styles this way, pass a `className` prop to `Alerts` from GUI.  Then the `Alerts` component can combine the class names passed in from props, and its own.

E.g. `<Box className={classNames(styles.alertsContainer, props.className)}> ... </Box>`

Reply:
Ah, got it! Thank you.